x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/configs/index.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiCodeBlock,
+  EuiSpacer,
+  EuiSkeletonText,
+  EuiCallOut,
+  EuiLink,
+  EuiCode,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { PackageInfo } from '../../../../../types';
+
+import { useGetInputsTemplatesQuery, useStartServices } from '../../../../../hooks';
+
+interface ConfigsProps {
+  packageInfo: PackageInfo;
+}
+
+export const Configs: React.FC<ConfigsProps> = ({ packageInfo }) => {
+  const { notifications, docLinks } = useStartServices();
+  const pkgName = packageInfo.name;
+  const pkgVersion = packageInfo.version;
```

Comment:
If we're going to use `title` below as suggested we could destructure everything at once, e.g. 

```suggestion
  const { name: pkgName, version: pkgVersion, title: pkgTitle } = packageInfo
```